src/parser/shaman/elemental/modules/talents/Ascendance.js

```diff
@@ -47,24 +90,42 @@ class Ascendance extends Analyzer {
   }
 
   statistic() {
-    return (
-      <StatisticBox
-        icon={<SpellIcon id={SPELLS.ASCENDANCE_TALENT_ELEMENTAL.id} />}
-        value={`${this.numCasts[SPELLS.LAVA_BURST.id]}`}
-        label="Lava Burst casts"
-        tooltip={`With a uptime of: ${formatPercentage(this.AscendanceUptime)} %<br>
+    let EBtooltip = "";
+    if(this.selectedCombatant.hasTalent(SPELLS.ELEMENTAL_BLAST_TALENT.id)) {
+      EBtooltip = <li>Elemental Blast: ${this.numCasts[SPELLS.ELEMENTAL_BLAST_TALENT.id]}</li>;
+    }
+    const tooltip = `With a uptime of: ${formatPercentage(this.AscendanceUptime)} %<br>
         Casts while Ascendance was up:<br>
         <ul>
           <li>Earth Shock: ${this.numCasts[SPELLS.EARTH_SHOCK.id]}</li>
-          <li>Elemental Blast: ${this.numCasts[SPELLS.ELEMENTAL_BLAST_TALENT.id]}</li>
-          <li>Flame Shock: ${this.numCasts[SPELLS.FLAME_SHOCK.id]}</li>
-          <li>Lightning Bolt: ${this.numCasts[SPELLS.LIGHTNING_BOLT.id]}</li>
+          <li>Lava Burst: ${this.numCasts[SPELLS.LAVA_BURST.id]}</li>
+          ${EBtooltip}
           <li>Other Spells: ${this.numCasts.others}</li>
         </ul>
-        `}
+        `;
+
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.ASCENDANCE_TALENT_ELEMENTAL.id} />}
+        value={`${this.numCasts[SPELLS.LAVA_BURST.id]}`}
+        label="Lava Burst casts"
+        tooltip={tooltip}
       />
     );
   }
+
+  suggestions(when) {
+    const abilities = `Lava Burst${this.selectedCombatant.hasTalent(SPELLS.ELEMENTAL_BLAST_TALENT.id) ? `, Elemental Blast ` : ``} and Earth Shock`;
+    when(this.numCasts.others).isGreaterThan(0)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span> Maximize your damage during ascendence by not casting weak spells.</span>)
```

Comment:
I'd suggest adding the abilities that you're supposed to cast to this text aswell, otherwise "other spells" is mentioned before the good spells.